src/Microsoft.CodeAnalysis.Analyzers/Core/MetaAnalyzers/DiagnosticDescriptorCreationAnalyzer.cs

```diff
@@ -276,28 +269,57 @@ public override void Initialize(AnalysisContext context)
             });
         }
 
-        private static void AnalyzeTitle(OperationAnalysisContext operationAnalysisContext, IObjectCreationOperation objectCreation)
+        private static bool TryGetDescriptorCreateMethodAndArguments(
+            IFieldInitializerOperation fieldInitializer,
+            INamedTypeSymbol diagnosticDescriptorType,
+            out IMethodSymbol creationMethod,
```

Comment:
```suggestion
            [NotNullWhen(true)] out IMethodSymbol? creationMethod,
```

Reply:
This file is not yet nullable enabled.

Reply:
I thought the whole repository was?

Reply:
Oh yes, sorry. Switching between Roslyn and RoslynAnalyzers repo confused me. Let me fix this in a follow-up PR.